### PR TITLE
Update hwdb and news for v250

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: tools/chromiumos/gen_autosuspend_rules.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python3
-# SPDX-License-Identifier: BSD-3-Clause
```
Comment (bluca) [inline tools/chromiumos/gen_autosuspend_rules.py]: What's this change for?

Comment (yuwata) [reply tools/chromiumos/gen_autosuspend_rules.py]: Not sure, the command `meson compile -C build update-hwdb update-hwdb-autosuspend` removes it.

Comment (yuwata) [reply tools/chromiumos/gen_autosuspend_rules.py]: The .py file is downloaded from https://chromium.googlesource.com/chromiumos/platform2/+/master/power_manager/udev/gen_autosuspend_rules.py?format=TEXT

Comment (keszybz) [reply tools/chromiumos/gen_autosuspend_rules.py]: Please exclude this change from the commit.

Comment (yuwata) [reply tools/chromiumos/gen_autosuspend_rules.py]: Hm, we may need to adjust `update-hwdb-autosuspend.sh`. In this time. I will just drop this and below changes.

Comment (yuwata) [reply tools/chromiumos/gen_autosuspend_rules.py]: > Please exclude this change from the commit.

Sure. Done.

Comment (keszybz) [reply tools/chromiumos/gen_autosuspend_rules.py]: It'd be nicer to submit *our* change as a patch for the upstream project.

**Review by bluca — APPROVED** [review]

**Pull request merged**